gef.py

```diff
@@ -3895,9 +3908,10 @@ def parse_string_range(s: str) -> Iterator[int]:
     return map(lambda x: int(x, 16), addrs)
 
 
-def gef_get_pie_breakpoint(num: int):  # -> PieVirtualBreakpoint
+def gef_get_pie_breakpoint(num: int) -> "PieVirtualBreakpoint":
```

Comment:
Unrelated to your PR, but you can mark it as deprecated?
```suggestion
@deprecated("Use `gef.session.pie_breakpoints[num]`")
def gef_get_pie_breakpoint(num: int) -> "PieVirtualBreakpoint":
```